apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-thumbnail-presenter.tsx

```diff
@@ -0,0 +1,28 @@
+import { IMultilingualText } from '@coscrad/api-interfaces';
+import { Card, CardContent, CardHeader } from '@mui/material';
+import { ReactNode } from 'react';
+import { MultilingualTextPresenter } from '../multilingual-text-presenter';
+
+export interface ResourceDetailThumbnailPresenterProps {
+    imageUrl?: string;
+    videoUrl?: string;
+    audioUrl?: string;
+    name: IMultilingualText;
+    children: ReactNode;
+}
+
+export const ResourceDetailThumbnailPresenter = ({
+    name,
+    audioUrl,
+    children,
+}: ResourceDetailThumbnailPresenterProps): JSX.Element => (
+    <Card>
+        <CardHeader>
+            <MultilingualTextPresenter text={name} />
+        </CardHeader>
+        {/* <CardMedia>
```

Comment:
Remove log

Reply:
Same as above for the commented out code.

Reply:
done